subprojects/model-core/src/main/java/org/gradle/api/internal/provider/AbstractMinimalProvider.java

```diff
@@ -116,7 +116,14 @@ public T getOrElse(T defaultValue) {
     @Deprecated
     @Override
     public final Provider<T> forUseAtConfigurationTime() {
-        // TODO:configuration-cache nag user
+        /*
+ TODO:configuration-cache start nagging in Gradle 8.x
+        DeprecationLogger.deprecateMethod(Provider.class, "forUseAtConfigurationTime")
+            .withAdvice("Simply remove the call.")
+            .willBeRemovedInGradle9()
+            .withUpgradeGuideSection(7, "for_use_at_configuration_time_deprecation")
+            .nagUser();
+*/
```

Comment:
:x: I would not leave the code commented but create an issue assigned to 8.0 instead